packages/protocol/migrationsConfig.js

```diff
@@ -80,6 +80,10 @@ const DefaultConfig = {
     participationBaselineUpdateFactor: 1 / 5,
     participationBaselineQuorumFactor: 1,
   },
+  governanceApproverMultiSig: {
+    signatories: [0x5409ed021d9299bf6814279a6a1411a7e866a631],
```

Comment:
nit: If tests would pass with this as an empty list, consider leaving it empty with a comment that says it needs to be overridden.

Ditto for the spender.

Reply:
And either way actually, please leave a comment that indicates that a 1/1 multisig is just a placeholder until the correct signatory addresses are known.

Reply:
i won't be able to do this with the current implementation -- there is validation in the multiSig that confirms that the signatory count is greater than 0, the required confirmations is greater than 0, and that the required confirmations is less than the signatory count.  

unless we wanted to remove this validation in the MultiSig contract...

I will add a comment regardless.